code-editor/auto-stop-idle/README.md

```diff
@@ -50,6 +50,45 @@ aws sagemaker update-domain \
     }'
 
 ```
+## Steps for installing in Internet-Free VPC environments:
+
+* Note : Your domain execution role must have privileges to download the packages from your s3 bucket where the below packages are uploaded. 
+
+1. Downloading autostop idle Python package and upload to s3 bucket.
+```
+ASI_VERSION=0.3.0
+PYTHON_PACKAGE=sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz
+echo "Downloading autostop idle Python package..."
+curl -LO https://github.com/aws-samples/sagemaker-studio-apps-lifecycle-config-examples/releases/download/v$ASI_VERSION/$PYTHON_PACKAGE
+aws s3 cp sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz <YOUR-S3-BUCKET-PATH> 
+```
+
+2. Edit the on-start.sh file and replace "45 and 46" with:
```

Comment:
> replace "45 and 46" 
This is easy to be obsolete after we further change the script and it would be hard to remember to update this.